index.js

```diff
@@ -200,3 +253,25 @@ formEl.addEventListener('submit', (e) => {
 
   checkInput();
 });
+
+const userData = JSON.parse(localStorage.getItem('userInfo'));
+
+if (userData) {
+  nameEl.value = userData.name;
+  emailEl.value = userData.email;
+  textEl.value = userData.text;
+}
+
+formEl.addEventListener('submit', () => {
+  const nameData = nameEl.value;
+  const emailData = emailEl.value;
+  const textData = textEl.value;
+
+  const userInfo = {
+    name: nameData,
+    email: emailData,
+    text: textData,
+  };
+
+  localStorage.setItem('userInfo', JSON.stringify(userInfo));
+});
```

Comment:
Good job so far, but there are a few missing points in your implementation of storing data on local storage;

- [ ] It's required that, when any of the input fields' values changed, the value is stored in localStorage instantly and then to the next input field and to the final one. In a sense, the values are expected to be stored in localStorage even before the form is submitted. That way, when the form data is submitted and the user goes back to the page and refreshes, the persisted data is fetched and present in the fields, and also, if a value is updated, that value that's updated is again updated on the localStorage.  Kindly implement that 👍🏽

**Below are the requirements snapshots**

<img width="932" alt="Screenshot 2023-02-02 at 10 31 58 AM" src="https://user-images.githubusercontent.com/50941074/216302628-3175d4ab-b69f-45ab-851c-4158c085f0b5.png">

<img width="685" alt="Screenshot 2023-02-02 at 10 40 21 AM" src="https://user-images.githubusercontent.com/50941074/216302898-e7d2e553-8499-419a-bae5-8077536ea3d8.png">